workshops/sprig/es-xl.md

```diff
@@ -0,0 +1,122 @@
+
+## Demo (2 minutos)
+
+Da una demostración rápida de un juego relativamente completo con código accesible. Puedes ir a la [galería](https://sprig.hackclub.com/gallery) y filtrar los juegos con la etiqueta de `beginner`.
+
+<img width="500" alt="Screen Shot 2022-09-23 at 5 00 24 PM" src="https://user-images.githubusercontent.com/27078897/192056026-c4c5492d-00e8-44fb-be35-6b74a1c3bfe2.png">
+
+Aquí hay algunos juegos que cumplen los criterios:
+
+- [Sokoban](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban.js)
+- [Penguin Slide](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/penguin_slide.js)
+- [Nomis](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/nomis.js)
+
+## Kit de herramientas (3 minutos)
+
+Abre el kit de herramientas y describe rápidamente las diferentes cosas que puedes hacer con estas. No se trata de explicar todas las funciones. Es solo para dar a la gente una idea de lo que es posible y dónde buscar ayuda. Dedica solo unos minutos a esto. Echa un vistazo al [fragmento de abajo](#toolkit) para examinar esencialmente cada función en Sprig.
+
+Asegúrate de **decirle a la gente cómo guardar los archivos**.
+
+Para guardar, añada un nombre así:
+
+```js
+/*
+@title: my_game_name
+@author: leo_mcelroy
+*/
+```
+
+Y luego pulsa `save`.
+
+Los archivos recientes están disponibles en el menú. Un asterisco junto al archivo significa que los cambios actuales no se han guardado. Los archivos se guardan en el almacenamiento local de su navegador. Solo serán accesibles en el ordenador y en el navegador que estés usando. **La forma más fiable de guardar es descargar el archivo `js`**.
+
+## ¡Dile a la gente que se ponga a hackear! (en el resto del tiempo disponible del taller)
+
+Dígale a la gente que empiece por trabajar con el [tutorial](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/getting_started.js).
+
+Una vez que la gente haya completado el tutorial, anímala a que empiece a pensar en su propio juego. A menudo, una buena forma de crear un juego original es empezar con uno ya existente y añadirle una nueva mecánica. Echa un vistazo a los juegos etiquetados como "hackeables" para encontrar bases remezclables.
+
+## Compartir las creaciones (10 minutos)
+
+Llama la atención de todos y pídeles que compartan sus juegos contigo. Pregunta si alguien quiere hacer una demostración de su juego al grupo. Celebra el trabajo de todos y aprovecha para que cada persona reciba comentarios, inspiración y afirmación.
+
+Si los juegos están completos, entonces [hacer un Pull Request](https://sprig.hackclub.com/share) para presentarlos a la galería y [ganar una consola Sprig](https://sprig.hackclub.com). 
```

Comment:
I would suggest to change `hacer un Pull Request` with `puedes hacer un Pull Request`.

Reply:
Everything else looks good!

Reply:
Ohh yes, i missed that